.circleci/config.yml

```diff
@@ -457,6 +457,39 @@ jobs:
             cd code
             yarn coverage
 
+## new workflow
+  sandboxes:
+    executor:
+      class: medium+
+      name: sb_node_14_browsers
+    # parallelism: 15
+    steps:
+      - git-shallow-clone/checkout_advanced:
+          clone_options: '--depth 1 --verbose'
+      - attach_workspace:
+          at: .
+      - run:
+          name: Running local registry
+          command: |
+            cd code
+            yarn local-registry --port 6000 --open
+          background: true
+      - run:
+          name: Wait for registry
+          command: |
+            cd code
+            yarn wait-on http://localhost:6000
```

Comment:
Should this not just be part of the multiplex-templates command?

Reply:
It's only necessary when first building the sandboxes. For latter multiplex commands (for instance running chromatic against every sandbox) we won't need verdaccio running